tools/checkcomments/checkcomments.go

```diff
@@ -31,7 +31,8 @@ import (
 )
 
 // todoRE represents correct "// TODO" comment format.
-var todoRE = regexp.MustCompile(`^// TODO (\Qhttps://github.com/FerretDB/\E[-\w]+/issues/\d+)$`)
+// It returns url and owner/repo name as submatches.
+var todoRE = regexp.MustCompile(`^// TODO (\Qhttps://github.com/\E(FerretDB/[-\w]+|microsoft/documentdb)/issues/\d+)$`)
```

Comment:
I think that's a bit too complicated 😬 I think we could just check the org name and ignore repository name. So this code would also check issues in other MS repos – but it is highly unlikely we will have any references to them

Reply:
Now checking any MS issues :+1: